<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceChargeCalculation</class>
 <widget class="QWidget" name="ServiceChargeCalculation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServiceChargeCalculation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>商品价值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数量</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>商品卖价</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>商人</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="Therapist">
              <property name="text">
               <string>Therapist</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="Ragman">
              <property name="text">
               <string>Ragman</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="Jaeger">
              <property name="text">
               <string>Jaeger</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QRadioButton" name="Mechanic">
              <property name="text">
               <string>Mechanic</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="Prapor">
              <property name="text">
               <string>Prapor</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="Peacekeeper">
              <property name="text">
               <string>Peacekeeper</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QRadioButton" name="Skier">
              <property name="text">
               <string>Skier</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QRadioButton" name="Fence">
              <property name="text">
               <string>Fence</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>3级情报中心</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>藏身处管理等级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>所有待售物品的需求</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>117</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="calculateButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background:#3949ab;
	border-radius:15px;
	color:rgb(247,247,247);
}

QPushButton:hover{
	background:#283593;
}

QPushButton:pressed{
	background:#283593;
}</string>
           </property>
           <property name="text">
            <string>计算</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>116</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>117</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="resultLabel">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#ff5722;
border-radius:15px;
color:rgb(247,247,247);</string>
           </property>
           <property name="text">
            <string>计算结果</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>116</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
